airbyte-integrations/bases/debezium-v1-9-6/src/main/java/io/airbyte/integrations/debezium/internals/DebeziumRecordIterator.java

```diff
@@ -61,8 +64,17 @@ public DebeziumRecordIterator(final LinkedBlockingQueue<ChangeEvent<String, Stri
     this.receivedFirstRecord = false;
     this.hasSnapshotFinished = true;
     this.signalledClose = false;
+    tsLastHeartbeat = null;
+    lastHeartbeatPosition = null;
   }
 
+  // #18987 The following logic incorporates heartbeat (CDC postgres only for now):
```

Comment:
tiny nit : Could you remove the reference to the #18987 (what is this exactly)?

Reply:
done